### ISSUE TITLE
option to disable "overscroll moves to next/previous file" behavior

### ISSUE THREAD

**Issue by pkra** [edrlab/thorium-reader#1938]
When setting Layout to "scrollable", Thorium opens the next file in the spine if users scroll down while already being at the end of the current file. Similarly, it opens the previous file, when scrolling up from the start of a file. 

When looking ahead in a file (e.g., "I want to scroll down to see how much longer this chapter is", "I want to scroll up to check I'm in the right chapter" ), I frequently trigger this accidentally. The resulting UX is quite jarring, especially if the files are larger and switching back and forth takes a long time.

It would be great if Thorium had a user option to disable this behavior, leaving the left/right buttons at the bottom to still take users to the previous/next chapter.

**Comment by danielweck** [comment]
based on my personal experience, I also feel that the overscroll-triggers-previous-or-next-document can be too sensitive, especially with Apple touchpad hardware / software (kinetic events). I tried to fine-tune the threshold used in the event handler to filter out some "mouse wheel" deltas, but the user experience remains inconsistent and confusing (the best way I can describe it is as if there was some sort of interaction hysteresis).

furthermore, personally I prefer when I reach the scroll beginning or end of a document against a hard stop, I don't like the supposedly seamless transition to the previous / next document. that's just my personal taste, but I'm probably not alone :)

in summary, I agree that it would be nice to be able to opt-out of this default feature ... or maybe even make it an opt-in?

**Comment by pkra** [comment]
> personally I prefer when I reach the scroll beginning or end of a document against a hard stop

I feel the same way but I understand if user research gave a different result.
